docs/src/_guide/troubleshooting.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Troubleshooting
+description: Learn what you can do if you need to troubleshoot
+layout: doc
+weight: 25
+---
+
+If you have a question about Meltano, are having trouble getting it to work, or have any kind of feedback, you can:
+
+<div class="columns is-multiline is-centered">
+    <div class="column is-half">
+      <div class="card chiclet">
+        <div class="card-content has-text-centered">
+          <div class="block">
+            <p><span class="icon is-large"><i class="fab fa-github fa-5x"></i></span></p>
+            <p>Check out the <a href="https://github.com/meltano/meltano/issues/">Meltano issue tracker</a> to see if someone else has already reported the same issue or made the same request. Feel free to weigh in with extra information, or just to make sure the issue gets the attention it deserves.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="column is-half">
+      <div class="card chiclet">
+        <div class="card-content has-text-centered">
+          <div class="block">
+            <p><span class="icon is-large"><i class="fab fa-slack fa-5x"></i></span></p>
```

Comment:
I think the images can be deleted - from what I can tell based on the docs homepage, just the favicon should work.